comps/llms/text-generation/predictionguard/README.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+
+[Prediction Guard](https://docs.predictionguard.com) allows you to utilize hosted open access LLMs, LVMs, and embedding functionality with seamlessly integrated safeguards. In addition to providing a scalable access to open models, Prediction Guard allows you to configure factual consistency checks, toxicity filters, PII filters, and prompt injection blocking. Join the [Prediction Guard Discord channel](https://discord.gg/TFHgnhAFKd) and request an API key to get started.
+
+# Get Started
+
+## Build Docker Image
+
+```bash
+cd ../../..
+docker build -t opea/llm-predictionguard:latest -f comps/llms/text-generation/predictionguard/docker/Dockerfile .
```

Comment:
pls note the docker image naming convention doesn't comply with the one we defined here

https://github.com/opea-project/docs/pull/107/files#diff-882936939f1eccccbb328c96070b53ef6c52b92f8403acb7f17618c368f47f67R92 line 92.